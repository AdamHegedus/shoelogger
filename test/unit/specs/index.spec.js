import Vue from 'vue';
import Index from '@/components/index';

describe('index.vue', () => {
    const Constructor = Vue.extend(Index);
    const vm = new Constructor().$mount();

    it('should match snapshot', () => {
        // ASSIGN

        // ACT
        const actual = vm.$el;

        // ASSERT
        expect(actual).toMatchSnapshot();
    });
});
