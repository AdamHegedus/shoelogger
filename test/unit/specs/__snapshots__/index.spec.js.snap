// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue should match snapshot 1`] = `
<div
  class="container"
>
  <div
    class="row"
  >
    <div
      class="col-sm col-lg-6"
    >
      <h1>
        Shoe Logger
      </h1>
       
      <p>
        A shoe wear management tool.
      </p>
    </div>
  </div>
</div>
`;
